### PR TITLE
Initial commit of DataFederations config for HCC VO

### PR DESCRIPTION
**Pull request by jthiltges**
Related to development discussion in SOFTWARE-3468

### CODE REVIEW

**Comment by bbockelm** [comment]
@matyasselmeci - note that the schema evolved a bit between #299 and this one.  In #299, we called the list of directories {{Authorizations}} while here we call them {{Namespaces}}.

I like {{Namespaces}} better as the {{AllowedCaches}} list is also an authorization; {{Namespaces}} feels more precise.

File: src/webapp/vos_data.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ def get_tree(self, authorized=False, filters: Filters = None) -> Dict:
         for vo_name in sorted(self.vos.keys(), key=lambda x: x.lower()):
             try:
                 expanded_vo_data = self._expand_vo(vo_name, authorized=authorized, filters=filters)
+
+                # Remove DataFederations block from VOSummary
+                expanded_vo_data.pop('DataFederations', None)
```
Comment (matyasselmeci) [inline src/webapp/vos_data.py]: This needs to be inside the `if expanded_vo_data` block to avoid a potential TypeError if expanded_vo_data is None.

Comment (bbockelm) [reply src/webapp/vos_data.py]: Got this fixed in #299 also.  Just removed this hunk.

**Comment by bbockelm** [comment]
I can confirm that, with these changes, we see the `/hcc` FQAN authorized to access `/hcc/focusday` and the Brian Bockelman DN can access both HCC and LIGO's areas:
```
$ PYTHONPATH=src ./bin/osg-authfile 2>/dev/null | grep hcc
g /hcc /hcc/focusday rl
u 6fb7593d.0 /hcc/focusday rl /user/ligo rl
```
@matyasselmeci - can you re-review?  I think this is ready to go.

**Pull request merged**